### ISSUE TITLE
Testcases testEntrypoint / testIn / testIntegerFunctions failing on ppc64 architecture

### ISSUE THREAD

**Issue by xambroz** [VirusTotal/yara-python#25]
The test-case testEntrypoint is failing on ppc64 architecture. Possibly some endianness issue with yara on ppc64 architecture. Tested with yara 3.5.0.

https://kojipkgs.fedoraproject.org//work/tasks/591/15270591/build.log

```
+ PYTHONPATH=/builddir/build/BUILDROOT/python-yara-3.5.0-5.el7.ppc64//usr/lib64/python2.7/site-packages/
+ /usr/bin/nosetests-2.7 -v '--exclude=^testModuleData$'
testAnonymousStrings (tests.TestYara) ... ok
testArithmeticOperators (tests.TestYara) ... ok
testAt (tests.TestYara) ... ok
testBitwiseOperators (tests.TestYara) ... ok
testBooleanOperators (tests.TestYara) ... ok
testCallback (tests.TestYara) ... ok
testComments (tests.TestYara) ... ok
testCompare (tests.TestYara) ... ok
testComparisonOperators (tests.TestYara) ... ok
testCompileFile (tests.TestYara) ... ok
testCompileFiles (tests.TestYara) ... ok
testCount (tests.TestYara) ... ok
testEntrypoint (tests.TestYara) ... FAIL
testExternals (tests.TestYara) ... ok
testFilesize (tests.TestYara) ... ok
testFor (tests.TestYara) ... ok
testHexStrings (tests.TestYara) ... ok
testIn (tests.TestYara) ... FAIL
testIncludeFiles (tests.TestYara) ... ok
testIntegerFunctions (tests.TestYara) ... FAIL
testLength (tests.TestYara) ... ok
testModules (tests.TestYara) ... ok
testOf (tests.TestYara) ... ok
testOffset (tests.TestYara) ... ok
testRE (tests.TestYara) ... ok
testStringIO (tests.TestYara) ... ok
testStrings (tests.TestYara) ... ok
testSyntax (tests.TestYara) ... ok
testWildcardStrings (tests.TestYara) ... ok
======================================================================
FAIL: testEntrypoint (tests.TestYara)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/yara-python-9fd9fd290872e36360e5e3839c49e21a908bf128/tests.py", line 681, in testEntrypoint
    ], PE32_FILE)
  File "/builddir/build/BUILD/yara-python-9fd9fd290872e36360e5e3839c49e21a908bf128/tests.py", line 274, in assertTrueRules
    self.assertTrue(r.match(data=data))
AssertionError: [] is not true
======================================================================
FAIL: testIn (tests.TestYara)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/yara-python-9fd9fd290872e36360e5e3839c49e21a908bf128/tests.py", line 580, in testIn
    ], PE32_FILE)
  File "/builddir/build/BUILD/yara-python-9fd9fd290872e36360e5e3839c49e21a908bf128/tests.py", line 274, in assertTrueRules
    self.assertTrue(r.match(data=data))
AssertionError: [] is not true
======================================================================
FAIL: testIntegerFunctions (tests.TestYara)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/yara-python-9fd9fd290872e36360e5e3839c49e21a908bf128/tests.py", line 898, in testIntegerFunctions
    ], b'\xAA\xBB\xCC\xDD')
  File "/builddir/build/BUILD/yara-python-9fd9fd290872e36360e5e3839c49e21a908bf128/tests.py", line 274, in assertTrueRules
    self.assertTrue(r.match(data=data))
AssertionError: [] is not true
----------------------------------------------------------------------
Ran 29 tests in 0.162s
FAILED (failures=3)
```

**Comment by xambroz** [comment]
Failing also on s390 architecture
http://s390.koji.fedoraproject.org/kojifiles/work/tasks/9135/2399135/build.log

```
Mock Version: 1.2.20
Mock Version: 1.2.20
ENTER ['do'](['bash', '--login', '-c', '/usr/bin/rpmbuild -bs --target s390x --nodeps /builddir/build/SPECS/python-yara.spec'], user='mockbuild'uid=995env={'TERM': 'vt100', 'LANG': 'en_US.UTF-8', 'PS1': '<mock-chroot> \\s-\\v\\$ ', 'HOME': '/builddir', 'SHELL': '/bin/bash', 'PROMPT_COMMAND': 'printf "\\033]0;<mock-chroot>\\007"', 'HOSTNAME': 'mock', 'PATH': '/usr/bin:/bin:/usr/sbin:/sbin'}chrootPath='/var/lib/mock/SHADOWBUILD-f26-build-624832-417976/root'gid=425shell=Falselogger=<mockbuild.trace_decorator.getLog object at 0x3ff7dc51eb8>timeout=86400printOutput=False)
Executing command: ['bash', '--login', '-c', '/usr/bin/rpmbuild -bs --target s390x --nodeps /builddir/build/SPECS/python-yara.spec'] with env {'TERM': 'vt100', 'LANG': 'en_US.UTF-8', 'PS1': '<mock-chroot> \\s-\\v\\$ ', 'HOME': '/builddir', 'SHELL': '/bin/bash', 'PROMPT_COMMAND': 'printf "\\033]0;<mock-chroot>\\007"', 'HOSTNAME': 'mock', 'PATH': '/usr/bin:/bin:/usr/sbin:/sbin'} and shell False
warning: bogus date in %changelog: Mon Aug 16 2016 Michal Ambroz <rebus at, seznam.cz> - 3.5.0-5
warning: Could not canonicalize hostname: fedora1.s390.bos.redhat.com
Building target platforms: s390x
Building for target s390x
Wrote: /builddir/build/SRPMS/python-yara-3.5.0-5.fc26.src.rpm
Child return code was: 0
ENTER ['do'](['bash', '--login', '-c', '/usr/bin/rpmbuild -bb --target s390x --nodeps /builddir/build/SPECS/python-yara.spec'], private_network=Trueuser='mockbuild'uid=995env={'TERM': 'vt100', 'LANG': 'en_US.UTF-8', 'PS1': '<mock-chroot> \\s-\\v\\$ ', 'HOME': '/builddir', 'SHELL': '/bin/bash', 'PROMPT_COMMAND': 'printf "\\033]0;<mock-chroot>\\007"', 'HOSTNAME': 'mock', 'PATH': '/usr/bin:/bin:/usr/sbin:/sbin'}gid=425chrootPath='/var/lib/mock/SHADOWBUILD-f26-build-624832-417976/root'shell=Falselogger=<mockbuild.trace_decorator.getLog object at 0x3ff7dc51eb8>timeout=86400printOutput=False)
Executing command: ['bash', '--login', '-c', '/usr/bin/rpmbuild -bb --target s390x --nodeps /builddir/build/SPECS/python-yara.spec'] with env {'TERM': 'vt100', 'LANG': 'en_US.UTF-8', 'PS1': '<mock-chroot> \\s-\\v\\$ ', 'HOME': '/builddir', 'SHELL': '/bin/bash', 'PROMPT_COMMAND': 'printf "\\033]0;<mock-chroot>\\007"', 'HOSTNAME': 'mock', 'PATH': '/usr/bin:/bin:/usr/sbin:/sbin'} and shell False
warning: bogus date in %changelog: Mon Aug 16 2016 Michal Ambroz <rebus at, seznam.cz> - 3.5.0-5
Building target platforms: s390x
Building for target s390x
Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.kzppAl
+ umask 022
+ cd /builddir/build/BUILD
+ cd /builddir/build/BUILD
+ rm -rf yara-python-9fd9fd290872e36360e5e3839c49e21a908bf128
+ /usr/bin/tar -xof -
+ /usr/bin/gzip -dc /builddir/build/SOURCES/python-yara-3.5.0-9fd9fd2.tar.gz
+ STATUS=0
+ '[' 0 -ne 0 ']'
+ cd yara-python-9fd9fd290872e36360e5e3839c49e21a908bf128
+ /usr/bin/chmod -Rf a+rX,u+w,g-w,o-w .
+ exit 0
Executing(%build): /bin/sh -e /var/tmp/rpm-tmp.YbS7Pv
+ umask 022
+ cd /builddir/build/BUILD
+ cd yara-python-9fd9fd290872e36360e5e3839c49e21a908bf128
+ CFLAGS='-O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -march=z9-109 -mtune=z10'
+ /usr/bin/python2 setup.py build '--executable=/usr/bin/python2 -s' --dynamic-linking
running build
running build_ext
building 'yara' extension
creating build
creating build/temp.linux-s390x-2.7
gcc -pthread -fno-strict-aliasing -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -march=z9-109 -mtune=z10 -D_GNU_SOURCE -fPIC -fwrapv -DNDEBUG -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -march=z9-109 -mtune=z10 -D_GNU_SOURCE -fPIC -fwrapv -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -march=z9-109 -mtune=z10 -fPIC -DHAVE_MEMMEM=1 -Iyara/libyara/include -Iyara/libyara/ -I. -I/usr/include/python2.7 -c yara-python.c -o build/temp.linux-s390x-2.7/yara-python.o
creating build/lib.linux-s390x-2.7
gcc -pthread -shared -Wl,-z,relro -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -march=z9-109 -mtune=z10 build/temp.linux-s390x-2.7/yara-python.o -L/usr/lib64 -lyara -lpython2.7 -o build/lib.linux-s390x-2.7/yara.so
+ sleep 1
+ CFLAGS='-O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -march=z9-109 -mtune=z10'
+ /usr/bin/python3 setup.py build '--executable=/usr/bin/python3 -s' --dynamic-linking
running build
running build_ext
building 'yara' extension
creating build/temp.linux-s390x-3.5
gcc -pthread -DDYNAMIC_ANNOTATIONS_ENABLED=1 -DNDEBUG -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -march=z9-109 -mtune=z10 -D_GNU_SOURCE -fPIC -fwrapv -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -march=z9-109 -mtune=z10 -fPIC -DHAVE_MEMMEM=1 -Iyara/libyara/include -Iyara/libyara/ -I. -I/usr/include/python3.5m -c yara-python.c -o build/temp.linux-s390x-3.5/yara-python.o
creating build/lib.linux-s390x-3.5
gcc -pthread -shared -Wl,-z,relro -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -march=z9-109 -mtune=z10 build/temp.linux-s390x-3.5/yara-python.o -L/usr/lib64 -lyara -lpython3.5m -o build/lib.linux-s390x-3.5/yara.cpython-35m-s390x-linux-gnu.so
+ sleep 1
Executing(%install): /bin/sh -e /var/tmp/rpm-tmp.MTf8dH
+ exit 0
+ umask 022
+ cd /builddir/build/BUILD
+ '[' /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x '!=' / ']'
+ rm -rf /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x
++ dirname /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x
+ mkdir -p /builddir/build/BUILDROOT
+ mkdir /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x
+ cd yara-python-9fd9fd290872e36360e5e3839c49e21a908bf128
+ CFLAGS='-O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -march=z9-109 -mtune=z10'
+ /usr/bin/python2 setup.py install -O1 --skip-build --root /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x
running install
running install_lib
creating /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x/usr
creating /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x/usr/lib64
creating /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x/usr/lib64/python2.7
creating /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x/usr/lib64/python2.7/site-packages
copying build/lib.linux-s390x-2.7/yara.so -> /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x/usr/lib64/python2.7/site-packages
writing byte-compilation script '/tmp/tmpMkK1QP.py'
/usr/bin/python2 -O /tmp/tmpMkK1QP.py
removing /tmp/tmpMkK1QP.py
running install_egg_info
running egg_info
creating yara_python.egg-info
writing yara_python.egg-info/PKG-INFO
writing top-level names to yara_python.egg-info/top_level.txt
writing dependency_links to yara_python.egg-info/dependency_links.txt
writing manifest file 'yara_python.egg-info/SOURCES.txt'
reading manifest file 'yara_python.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'yara_python.egg-info/SOURCES.txt'
Copying yara_python.egg-info to /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x/usr/lib64/python2.7/site-packages/yara_python-3.5.0-py2.7.egg-info
running install_scripts
warning: no files found matching 'yara/libyara/modules/module_list'
warning: no files found matching '*.c' under directory 'yara'
warning: no files found matching '*.h' under directory 'yara'
+ CFLAGS='-O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -march=z9-109 -mtune=z10'
+ /usr/bin/python3 setup.py install -O1 --skip-build --root /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x
running install
running install_lib
creating /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x/usr/lib64/python3.5
creating /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x/usr/lib64/python3.5/site-packages
copying build/lib.linux-s390x-3.5/yara.cpython-35m-s390x-linux-gnu.so -> /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x/usr/lib64/python3.5/site-packages
writing byte-compilation script '/tmp/tmpk2m4t1w7.py'
/usr/bin/python3 -O /tmp/tmpk2m4t1w7.py
removing /tmp/tmpk2m4t1w7.py
running install_egg_info
running egg_info
writing yara_python.egg-info/PKG-INFO
writing dependency_links to yara_python.egg-info/dependency_links.txt
writing top-level names to yara_python.egg-info/top_level.txt
reading manifest file 'yara_python.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching 'yara/libyara/modules/module_list'
warning: no files found matching '*.c' under directory 'yara'
writing manifest file 'yara_python.egg-info/SOURCES.txt'
Copying yara_python.egg-info to /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x/usr/lib64/python3.5/site-packages/yara_python-3.5.0-py3.5.egg-info
running install_scripts
warning: no files found matching '*.h' under directory 'yara'
+ /usr/lib/rpm/find-debuginfo.sh --strict-build-id -m --run-dwz --dwz-low-mem-die-limit 10000000 --dwz-max-die-limit 50000000 /builddir/build/BUILD/yara-python-9fd9fd290872e36360e5e3839c49e21a908bf128
extracting debug info from /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x/usr/lib64/python2.7/site-packages/yara.so
extracting debug info from /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x/usr/lib64/python3.5/site-packages/yara.cpython-35m-s390x-linux-gnu.so
/usr/lib/rpm/sepdebugcrcfix: Updated 2 CRC32s, 0 CRC32s did match.
97 blocks
+ /usr/lib/rpm/check-buildroot
+ /usr/lib/rpm/brp-compress
+ /usr/lib/rpm/brp-strip-static-archive /usr/bin/strip
+ /usr/lib/rpm/brp-python-bytecompile /usr/bin/python 1
Bytecompiling .py files below /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x/usr/lib64/python2.7 using /usr/bin/python2.7
Bytecompiling .py files below /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x/usr/lib64/python3.5 using /usr/bin/python3.5
Bytecompiling .py files below /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x/usr/lib/debug/usr/lib64/python2.7 using /usr/bin/python2.7
Bytecompiling .py files below /builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x/usr/lib/debug/usr/lib64/python3.5 using /usr/bin/python3.5
+ /usr/lib/rpm/brp-python-hardlink
+ /usr/lib/rpm/redhat/brp-java-repack-jars
Executing(%check): /bin/sh -e /var/tmp/rpm-tmp.EQZOb0
+ umask 022
+ cd /builddir/build/BUILD
+ cd yara-python-9fd9fd290872e36360e5e3839c49e21a908bf128
++ ls /usr/bin/nosetests-2.7
+ NOSETESTS2=/usr/bin/nosetests-2.7
+ PYTHONPATH=/builddir/build/BUILDROOT/python-yara-3.5.0-5.fc26.s390x//usr/lib64/python2.7/site-packages/
+ /usr/bin/nosetests-2.7 -v '--exclude=^testModuleData$'
testAnonymousStrings (tests.TestYara) ... ok
testArithmeticOperators (tests.TestYara) ... ok
testAt (tests.TestYara) ... ok
testBitwiseOperators (tests.TestYara) ... ok
testBooleanOperators (tests.TestYara) ... ok
testCallback (tests.TestYara) ... ok
testComments (tests.TestYara) ... ok
testCompare (tests.TestYara) ... ok
testComparisonOperators (tests.TestYara) ... ok
testCompileFile (tests.TestYara) ... ok
testCompileFiles (tests.TestYara) ... ok
testCount (tests.TestYara) ... ok
testEntrypoint (tests.TestYara) ... FAIL
testExternals (tests.TestYara) ... ok
testFilesize (tests.TestYara) ... ok
testFor (tests.TestYara) ... ok
testHexStrings (tests.TestYara) ... ok
testIn (tests.TestYara) ... FAIL
testIncludeFiles (tests.TestYara) ... ok
testIntegerFunctions (tests.TestYara) ... FAIL
testLength (tests.TestYara) ... ok
testModules (tests.TestYara) ... ok
testOf (tests.TestYara) ... ok
testOffset (tests.TestYara) ... ok
testRE (tests.TestYara) ... ok
testStringIO (tests.TestYara) ... ok
testStrings (tests.TestYara) ... ok
testSyntax (tests.TestYara) ... ok
testWildcardStrings (tests.TestYara) ... ok
======================================================================
FAIL: testEntrypoint (tests.TestYara)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/yara-python-9fd9fd290872e36360e5e3839c49e21a908bf128/tests.py", line 681, in testEntrypoint
    ], PE32_FILE)
  File "/builddir/build/BUILD/yara-python-9fd9fd290872e36360e5e3839c49e21a908bf128/tests.py", line 274, in assertTrueRules
    self.assertTrue(r.match(data=data))
AssertionError: [] is not true
======================================================================
FAIL: testIn (tests.TestYara)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/yara-python-9fd9fd290872e36360e5e3839c49e21a908bf128/tests.py", line 580, in testIn
    ], PE32_FILE)
  File "/builddir/build/BUILD/yara-python-9fd9fd290872e36360e5e3839c49e21a908bf128/tests.py", line 274, in assertTrueRules
    self.assertTrue(r.match(data=data))
AssertionError: [] is not true
======================================================================
FAIL: testIntegerFunctions (tests.TestYara)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/yara-python-9fd9fd290872e36360e5e3839c49e21a908bf128/tests.py", line 898, in testIntegerFunctions
    ], b'\xAA\xBB\xCC\xDD')
  File "/builddir/build/BUILD/yara-python-9fd9fd290872e36360e5e3839c49e21a908bf128/tests.py", line 274, in assertTrueRules
    self.assertTrue(r.match(data=data))
AssertionError: [] is not true
----------------------------------------------------------------------
Ran 29 tests in 0.172s
FAILED (failures=3)
RPM build errors:
error: Bad exit status from /var/tmp/rpm-tmp.EQZOb0 (%check)
    bogus date in %changelog: Mon Aug 16 2016 Michal Ambroz <rebus at, seznam.cz> - 3.5.0-5
    Bad exit status from /var/tmp/rpm-tmp.EQZOb0 (%check)
Child return code was: 1
EXCEPTION: [Error()]
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/mockbuild/trace_decorator.py", line 88, in trace
    result = func(*args, **kw)
  File "/usr/lib/python3.5/site-packages/mockbuild/util.py", line 562, in do
    raise exception.Error("Command failed. See logs for output.\n # %s" % (command,), child.returncode)
mockbuild.exception.Error: Command failed. See logs for output.
 # bash --login -c /usr/bin/rpmbuild -bb --target s390x --nodeps /builddir/build/SPECS/python-yara.spec
```

**Comment by xambroz** [comment]
Seems to be fixed in 3.9.0
